backend/census_historical_migration/end_to_end_core.py

```diff
@@ -51,18 +51,16 @@
 
 
 def step_through_certifications(sac):
-    stati = [
-        SingleAuditChecklist.STATUS.IN_PROGRESS,
-        SingleAuditChecklist.STATUS.READY_FOR_CERTIFICATION,
-        SingleAuditChecklist.STATUS.AUDITOR_CERTIFIED,
-        SingleAuditChecklist.STATUS.AUDITEE_CERTIFIED,
-        SingleAuditChecklist.STATUS.CERTIFIED,
-        SingleAuditChecklist.STATUS.SUBMITTED,
-        SingleAuditChecklist.STATUS.DISSEMINATED,
-    ]
-    for status in stati:
-        sac.transition_name.append(status)
-        sac.transition_date.append(datetime.now(timezone.utc))
+    sac.transition_to_ready_for_certification()
+    sac.transition_to_auditor_certified()
+    sac.transition_to_auditee_certified()
+
+    # FIXME-MSHD: We have no method transition_to_certified()
+    sac.transition_name.append(SingleAuditChecklist.STATUS.CERTIFIED)
+    sac.transition_date.append(datetime.now(timezone.utc))
+
+    sac.transition_to_submitted()
+    sac.transition_to_disseminated()
     sac.save()
 
```

Comment:
These changes to state transitions are bringing back this transition error for tribal data.

ERROR Unexpected error type <class 'django_fsm.TransitionNotAllowed'>: Can't switch from state 'in_progress' using method 'transition_to_auditor_certified'
ERROR /src/census_historical_migration/end_to_end_core.py:298 run_end_to_end: step_through_certifications(sac)
ERROR /src/census_historical_migration/end_to_end_core.py:55 step_through_certifications: sac.transition_to_auditor_certified()
ERROR /usr/local/lib/python3.10/site-packages/django_fsm/init.py:573 _change_state: return fsm_meta.field.change_state(instance, func, *args, **kwargs)
ERROR /usr/local/lib/python3.10/site-packages/django_fsm/init.py:339 change_state: raise TransitionNotAllowed(